.gitignore

```diff
@@ -27,3 +27,7 @@ ccdaservice/node_modules
 
 # testing
 .phpunit.result.cache
+
+#ruthkonyn fork
+*/.directory
+
```

Comment:
hi @ruthkonyn , if the .gitignore was working then the .directory file wouldn't be listed below,

would prob remove it and undo the .gitignore change